src/ngx_http_auth_jwt_header_processing.c

```diff
@@ -16,80 +16,73 @@
  * Sample code from nginx.
  * https://www.nginx.com/resources/wiki/start/topics/examples/headers_management/?highlight=http%20settings
  */
-ngx_table_elt_t* search_headers_in(ngx_http_request_t *r, u_char *name, size_t len)
+ngx_table_elt_t *search_headers_in(ngx_http_request_t *r, u_char *name, size_t len)
 {
-	ngx_list_part_t            *part;
-	ngx_table_elt_t            *h;
-	ngx_uint_t                  i;
+  ngx_list_part_t *part;
+  ngx_table_elt_t *h;
+  ngx_uint_t i;
 
-	// Get the first part of the list. There is usual only one part.
-	part = &r->headers_in.headers.part;
-	h = part->elts;
+  // Get the first part of the list. There is usual only one part.
+  part = &r->headers_in.headers.part;
+  h = part->elts;
 
-	// Headers list array may consist of more than one part, so loop through all of it
-	for (i = 0; /* void */ ; i++)
-	{
-		if (i >= part->nelts)
-		{
-			if (part->next == NULL)
-			{
-				/* The last part, search is done. */
-				break;
-			}
+  // Headers list array may consist of more than one part, so loop through all of it
+  for (i = 0; /* void */; i++)
```

Comment:
Pre-increment is better for performance reason in a for loop
```suggestion
  for (i = 0; /* void */; ++i)
```